### PR TITLE
Use inspect module to determine plugin base directory

### PR DESCRIPTION
**Pull request by ronichoudhury**
This PR replaces a brittle mechanism (using the `subprocess` module to ask `girder-install` what the Girder plugin path is) with a more robust one that uses the `inspect` module's reflection capabilities to figure out where the calling module, and therefore the `resonant-laboratory` directory, is.

### CODE REVIEW

File: app/resonant-laboratory/server/pluginDir.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import inspect
+import os
+
+
+def pluginDir():
+    me = inspect.stack()[0][1]
```
Comment (jeffbaumes) [inline app/resonant-laboratory/server/pluginDir.py]: A comment might be warranted on this line to explain what it does.

**Comment by jeffbaumes** [comment]
LGTM other than suggestion to add a comment

**Comment by ronichoudhury** [comment]
@jeffbaumes can you check that comment to see if it's sufficient?

**Comment by jeffbaumes** [comment]
LGTM, thanks.

**Pull request merged**